pkg/daemon/daemon.go

```diff
@@ -558,10 +599,39 @@ func (dn *Daemon) applyNodePtpProfile(runID int, nodeProfile *ptpv1.PtpProfile)
 			printNodeProfile(nodeProfile)
 			return fmt.Errorf("failed to write the configuration file named %s: %v", configPath, err)
 		}
-		printNodeProfile(nodeProfile)
-		dn.processManager.process = append(dn.processManager.process, &dprocess)
```

Comment:
We're always doing this right, either on line 606 or 633?  Why is it necessary to handle like this?

Reply:
This change it to build phc2sys at the end of the list, so that phc2sys process starts at the end